G1ANT.Appium/Commands/AppiumGetTextFromElementsCommand.cs

```diff
@@ -9,12 +9,12 @@ public class GetTextFromElementsCommand : Language.Command
     {
         public class Arguments : CommandArguments
         {
-            [Argument(Tooltip = "Specify by wich capability the element will be searched")]
-            public TextStructure By { get; set; } = new TextStructure("");
-
             [Argument(Tooltip = "Provide name of the capaility")]
```

Comment:
Those arguments are used twice here and in "G1ANT.Appium/Commands/GetTextCommand.cs → G1ANT.Appium/Commands/AppiumGetTextCommand.cs"

Extract them to a base class.